prep-javascript-variables/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullname = 'Daniel Virrueta';
+    var numberofPets = 1;
+    var likesVegetables = (true);
+    var nothing = 'null';
```

Comment:
We need to leave off the quotes of `null` so that the javascript engine doesn't interpret it as a string.